.env.example

```diff
@@ -1,10 +0,0 @@
-DEVISE_JWT_SECRET_KEY=
```

Comment:
This file is an example of the env vars we have for this project, it cannot be deleted.

Reply:
Ok. Sorry for that.